Source/diablo.cpp

```diff
@@ -2228,6 +2228,70 @@ void InitPadmapActions()
 	    N_("Toggles if automap is displayed."),
 	    ControllerButton_BUTTON_LEFTSTICK,
 	    DoAutoMap);
+	options.Padmapper.AddAction(
+	    "Automap Move Up",

```

Comment:
This is the identifier that goes to the left of the `=` sign in the INI file. For consistency's sake, can we take the spaces out?